### PR TITLE
feat: order imports eslint configs

### PR DESCRIPTION
**Pull request by anamontiaga**
### Overview

Test to sort imports by path groups using an eslint plugin. We would need to align if this is the order we want and add all the dependencies

### Testing instructions

- [ ] The order is generated with the autosave.
- [ ]  It has not yet been implemented in the files. It will do it as we work on each file.

### CODE REVIEW

**Comment by anamontiaga** [comment]
This is how it fit on a file with many types of import. It is only a proposal:

 sort imports

File: app/.eslintrc.js [hunk 1]
```diff
@@ -31,5 +31,71 @@ module.exports = {
...
+            group: 'external',
+            position: 'before',
+          },
+          {
+            pattern: 'store/**',
+            group: 'external',
+          },
+          {
+            pattern: 'hooks/**',
+            group: 'unknown',
```
Comment (mbarrenechea) [inline app/.eslintrc.js:73]: why `hooks` folder belongs to `unkown` group?

Comment (mbarrenechea) [reply app/.eslintrc.js:73]: I don't like that internal imports come before hooks

image

**Review by mbarrenechea — CHANGES_REQUESTED** [review]

**Comment by anamontiaga** [comment]
Result after changes requested made:

order:imports fixes

File: app/.eslintrc.js (hunk 1 above)
Comment (anamontiaga) [reply app/.eslintrc.js]: Attached new proposal.

**Review by mbarrenechea — APPROVED** [review]
Let's see how it works

**Pull request merged**